### ISSUE TITLE
Webhooks: Can't use `stripe trigger checkout.session.async_payment_succeeded`

### ISSUE THREAD

**Issue by katherineqian** [stripe/stripe-cli#1112]
_The more information we have the easier it is for us to help. Feel free to remove any sections that might not apply_

### Issue
_Describe what happened and what you were trying to do_
I tried to use the Stripe CLI on web to test my webhook. 

### Expected Behavior
_Tell us what you expected to happen_
I expected to be able to trigger an async payment success event using `stripe trigger checkout.session.async_payment_succeeded`. 

### Steps to reproduce
_What are the steps we can take to reproduce this and verify it's fixed?_
Run `stripe trigger checkout.session.async_payment_succeeded` and successfully trigger an event. 

### Traceback
_Share any debug output that was given by the CLI_
```
{"error": {
"message":  "An error has occurred confirming the Checkout Session.",
"request_log_url":  ["https://dashboard.stripe.com/test/logs/req_uKwg65sLrDOSqd?t=1691871196"](https://dashboard.stripe.com/test/logs/req_uKwg65sLrDOSqd?t=1691871196),
"type":  "invalid_request_error", },
```

### Environment
_Select one of: macOS, Linux, Windows, Other_
Other (Chrome web)

**Comment by kyang-stripe** [comment]
Hi @katherineqian, what version of stripe-cli are you using? Could you update to the latest version and see if that fixes your issue?

**Comment by etsai-stripe** [comment]
hi @katherineqian 
i have checked with stripe CLI version 1.16.0, and it worked. could you please update and try again? i will close this ticket now

```
$  stripe trigger checkout.session.async_payment_succeeded
A newer version of the Stripe CLI is available, please update to: v1.17.1
Setting up fixture for: product
Running fixture for: product
Setting up fixture for: price
Running fixture for: price
Setting up fixture for: checkout_session
Running fixture for: checkout_session
Setting up fixture for: payment_page
Running fixture for: payment_page
Setting up fixture for: payment_method
Running fixture for: payment_method
Setting up fixture for: payment_page_confirm
Running fixture for: payment_page_confirm
Trigger succeeded! Check dashboard for event details.
$ stripe version
stripe version 1.16.0
A newer version of the Stripe CLI is available, please update to: v1.17.1
```
